tests/test_aws_node_group_actions.py

```
@@ -0,0 +1,158 @@
+    mock_boto_client.return_value = eks_mock
+
+    params = EksNodeGroupParams(
+        **{**BASE_PARAMS, "aws_access_key_id": "AKID", "aws_secret_access_key": "SECRET"}
+    )
+    eks_scale_node_group(mock_event, params)
+
+    mock_boto_client.assert_called_once_with(
+        "eks",
+        region_name="us-west-2",
+        aws_access_key_id="AKID",
+        aws_secret_access_key="SECRET",
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "aws_secret_access_key"

(S106)